citi-onboarding/Delta-odonto
# Hotfix/integration

## What I Did
- Integrated the Content section to the backend
- Fixed the Our Clients section arrows (although form what I've seen now they were fixed in another PR, oops!)
- Made the content nodemailer!

## How to test
- Run both the back and the frontend of the project
- Access the backend of the project on localhost:3001/admin
- Change the content on the Content model and see if the front-end will reflect those changes 
- Try to access one of Deltas contents, answer the questions on the modal and check if an e-mail with your data showed up on Deltas e-mail (Deltas e-mail is registered on the project .env file)
